resources/assets/js/helpers.js

```diff
@@ -103,6 +105,30 @@ $.extend(true, laravelValidation, {
 
 
         /**
+         * Return specified rule from element
+         *
+         * @param rule
+         * @param element
+         * @returns object
+         */
+        getLaravelValidation: function(rule, element) {
+
+            var found = undefined;
+            $.each($.validator.staticRules(element), function(key, rules) {
+                if (key=="laravelValidation") {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/albert-moreno-t/laravel-jsvalidation/file/842620831/issues/source?bid=2225045&fileBranchId=2805859#l118)